contracts/Strategy.sol

```diff
@@ -301,57 +199,8 @@ contract Strategy is BaseStrategy {
         }
     }
 
-    // NOTE: Can override `tendTrigger` and `harvestTrigger` if necessary
-
     function prepareMigration(address _newStrategy) internal override {
         liquidatePosition(uint256(-1)); //withdraw all. does not matter if we ask for too much
-        _sell();
-    }
-
-    //Use this to toggle emergencywithdraw from withdrawing via exit
-    function toggleExit() external onlyGuardians {
-        useExitForEmergency = !useExitForEmergency;
-    }
-
-    function enableApproval() external onlyGuardians {
-        want.safeApprove(staker, uint256(-1));
-    }
-
-    function disableApproval() external onlyGuardians {
-        want.safeApprove(staker, 0);
-    }
-
-    function emergencyWithdrawal() external  onlyGuardians {
-        uint256 balStake = balanceOfStake();
-        if(useExitForEmergency)
-            ISynthetixRewards(staker).exit();
-        else if (balStake > 0)
-            ISynthetixRewards(staker).withdraw(balStake);
-    }
```

Comment:
Might be useful to keep emergencywithdrawal code from the masterchef strat

Reply:
it doesn't exist in the popsicle contract.